src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -96,13 +101,15 @@ declare global {
 const DEFAULT_VIEW: View = "info";
 
 @customElement("ha-more-info-dialog")
-export class MoreInfoDialog extends LitElement {
+export class MoreInfoDialog extends ScrollableFadeMixin(LitElement) {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
   @property({ type: Boolean, reflect: true }) public large = false;
 
   @state() private _parentEntityIds: string[] = [];
 
+  @query(".content") private _contentElement!: HTMLDivElement;
```

Comment:
```suggestion
  @query(".content") private _contentElement?: HTMLDivElement;
```

it's not always there. If entity_id is missing it's not rendered!